conf/application.conf

```diff
@@ -225,6 +225,14 @@ microservice {
       port = 9508
       enabled = true
     }
+
+    upscan-initiate {
+      protocol = "http"
+      host = "localhost"
+      port = 9570
+      callbackUrl = "https://myservice.com/callback"
+      maximumFileSize = 22
```

Comment:
Can we add a comment in the config to make it clear that we're specifying this in megabytes, just so it's clear at one glance?

(I was kind of wondering if we should specify this in bytes or KB instead, megabytes is reasonable, as long as this accepts a decimal... may be better to be KB at least.)

Reply:
will add a comment to reflect it has 22 MB